### ISSUE TITLE
Why is self.stable_steps set as 100 when cur_level==6 in "train.py"?

### ISSUE THREAD

**Issue by SheldonTsui** [ss12f32v/GANsynth-pytorch#1]
1. Why do you set self.stable_and_fadein_step as [1,1,1,1,1,1,1] in "train.py", but self.stable_steps=100 when cur_level=6? Does this mean that previous training stages are NOT important? Because I found that the loss curves are normal and steady when cur_level < 6, and the loss will explode in the final training stage. 
2. There's no such phenomenon when training with the original GANSynth Magenta code. Could you provide your loss curves during training? 
3. Moreover, the audio samples generated by the model trained with your code are OK overall, but have many noises inside. Could you provide more training details? I am interested in this direction, and it's my pleasure to have more discussion with you.

**Comment by ss12f32v** [comment]
Hi SheldonTsui :

1. The [1,1,1,1,1,1,1]  is used for testing the whole training procedure is ok or not. In my experiment, the loss will also oscillate between 200 ~-200  when the cur_level reach to six. 
2. Loss curves in GAN don't mean much things due to the min-max game objective function, so I wrote a function to plot the image to observe the training result. (But when the loss enormously explodes, it may mean that one of the D or G has become too strong.) 
3. I have tried many times to train the whole dataset with the different settings (LR, stable and fade in steps, etc.). Actually, I can not reproduce the result that Google released. There are still some diversity and quality problems in my PyTorch model. I think magenta original GANSynth code is using some training tips, but they didn't mention on the ICLR paper.

Please feel free to discuss here, I am also a beginner in GAN.
